test/jdk/java/awt/regtesthelpers/PassFailJFrame.java

```diff
@@ -1310,14 +1319,34 @@ public static void forcePass() {
     }
 
     /**
-     *  Forcibly fail the test.
+     * Forcibly fail the test.
+     * <p>
+     * Use this method in semi-automatic tests when
+     * the test determines that all/any of the conditions for passing the test are not met.
```

Comment:
```suggestion
     * <p>
     * Use this method in semi-automatic tests when
     * it is determined that the conditions for passing the test cannot be met.
```
I still don't like this phrase… but I can't come up with a different one.